docs/proposals/emptydir-pv.md

```diff
@@ -0,0 +1,151 @@
+# Emptydir persistent volume
```

Comment:
Nit: It's easier for people to comment on docs if you manually wrap the lines, otherwise comments are at the granularity of paragraphs.